docs/getting-started/index.md

```diff
@@ -1,603 +1,220 @@
-Feel free to explore the [Kendo UI for Vue Data Grid documentation page](slug:overview_grid) to get a sense of just how many things the Grid can do.
-
-## 8. Add a Kendo UI for Vue Window
-The `products` array contains some fields which are not displayed in the Grid. In this section, you’ll use the Kendo UI for Vue Window to display those additional product details when users select a Grid row.
-
-Here are the required steps.
-First import the Window component:
+    ```
 
-```js
-import { Window } from '@progress/kendo-vue-dialogs';
+1. Define the Grid in the `<script setup>` configuration. Add user-friendly [column titles](slug:api_grid_gridcolumnprops#toc-title) by using the `title` property:
 
-export default {
-  components: {
-    //..............
-    'window': Window
-  },
-  //..............
-```
-
-Next, define new `windowVisible` and `gridClickedRow` data properties.
-```js
-setup() {
-    //..............
-    const gridClickedRow = ref({});
-    const windowVisible = ref(false);
-    //..............
-
-    return {
-      gridClickedRow, windowVisible ........
-    }
-}
-```
-Next, add a [row click handler](slug:api_grid_gridprops#toc-rowclick) to the Grid.
-```html
-<grid @rowclick="rowClick">
-    //..............
-</grid>
-```
-After that, add the `rowClick` function below, which will set the `windowVisible` flag to true, and assign the data item of the clicked row to the gridClickedRow property. You’ll use the data item values to render the Window's content.
-```js
-setup() {
-    //..............
-    const rowClick = (event: GridRowClickEvent) => {
-      windowVisible.value = true;
-      gridClickedRow.value = event.dataItem;
-    };
-    //..............
-
-    return {
-      rowClick, ........
-    }
-}
-```
-Next, add the following Window declaration. (Add it immediately after the Grid's definition in the template.) Notice how the Window will be rendered only if the `windowVisible` flag value is true.
-```html
-<window v-if="windowVisible" :title="'Product Details'" @close="closeWindow" :height="250">
-  <dl style="{textAlign:left}">
-    <dt>Product Name</dt>
-    <dd>{{gridClickedRow.ProductName}}</dd>
-    <dt>Product ID</dt>
-    <dd>{{gridClickedRow.ProductID}}</dd>
-    <dt>Quantity per Unit</dt>
-    <dd>{{gridClickedRow.QuantityPerUnit}}</dd>
-  </dl>
-</window>
-```
-Finally, add the following Window [close handler](slug:api_dialogs_windowprops#toc-close), which will set the `windowVisible` flag to false when the user closes the Window.
-
-```js
-setup() {
-    //..............
-    const closeWindow = () => {
-      windowVisible.value = false;
-    }
-    //..............
-
-    return {
-      closeWindow, ........
-    }
-}
-```
-With this code in place, try tapping on a row in the Grid. You should see a custom Window appear with additional product information.
+    ```js
+      <script setup>
```

Comment:
We can add the following snippet for importing the Grid:

export default defineComponent({
  components: {
    grid: Grid,
  },

Reply:
Added.